crates/ra_lsp_server/src/server_world.rs

```diff
@@ -73,7 +73,7 @@ impl ServerWorldState {
                     if let (Some(&from), Some(&to)) =
                         (sysroot_crates.get(&from), sysroot_crates.get(&to))
                     {
-                        if let Err(_) = crate_graph.add_dep(from, name.clone(), to) {
+                        if crate_graph.add_dep(from, name.clone(), to).is_err() {
```

Comment:
I prefer the former here: here we are doing a non-obvious thing, ignorring an error, and `_` makes that clearer.